frontend/shared/utils/tryWebAppStartup.js

```diff
@@ -7,10 +7,12 @@ export default function tryWebAppStartup() {
 
     If requirements are not present, noop.
      */
-    const el = document.getElementById("main"),
```

Comment:
This change was necessary for `startup` to work correctly on admin pages, since an element with id `main` already existed on those templates. However, subsequent commits to this PR effectively removes `startup` from admin pages, but I think this change should remain in just to make it more flexible. Let me know if you disagree and I'll revert it!

Reply:
this seems reasonable to me; I bet it'll come in handy...